backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/IGitMacheteRepositorySnapshot.java

```diff
@@ -21,6 +21,8 @@
 
   Option<IManagedBranchSnapshot> getManagedBranchByName(String branchName);
 
+  List<String> getDuplicatedBranchNames();
```

Comment:
Btw: crazy/unlikely corner case, what if a branch is both duplicated AND skipped? ;) that's perfectly possible ofc

Reply:
Also, pls add some 3-ish test cases to `backendImpl/src/tests`

Reply:
Presently it will be considered as a skipped one only. does it make sense?

Reply:
Okay, makes sense, actually maybe even more than the inverse (i.e. considered duplicate only)... but still, that's more of a puzzle than actual problem ;)